apps/main/src/dex/components/PageCreatePool/TokensInPool/SelectTokenButton.tsx

```diff
@@ -0,0 +1,289 @@
+import { CreateToken } from '@/dex/components/PageCreatePool/types'
+import { t } from '@ui-kit/lib/i18n'
+import { useButton } from '@react-aria/button'
+import { useFilter } from '@react-aria/i18n'
+import { useEffect, useMemo, useRef, useState } from 'react'
+import { useOverlayTriggerState } from '@react-stately/overlays'
+import styled from 'styled-components'
+import { delayAction, shortenTokenAddress } from '@/dex/utils'
+import useStore from '@/dex/store/useStore'
+import { STABLESWAP } from '@/dex/components/PageCreatePool/constants'
+import Box from '@ui/Box'
+import Button from '@ui/Button'
+import Spinner, { SpinnerWrapper } from '@ui/Spinner'
+import { TokenIcon } from '@ui-kit/shared/ui/TokenIcon'
+import { Chip } from '@ui/Typography'
+import Checkbox from '@ui/Checkbox'
+import { ChainId, CurveApi } from '@/dex/types/main.types'
+import { filterTokens } from '@ui-kit/utils'
+import { TokenSelectorModal } from '@ui-kit/features/select-token/ui/modal/TokenSelectorModal'
+
+type Props = {
+  curve: CurveApi
+  chainId: ChainId
+  disabledKeys?: string[]
+  haveSigner: boolean
+  blockchainId: string
+  selectedAddress: string
+  tokens: CreateToken[]
+  onSelectionChange: (selectedAddress: React.Key) => void
+}
+
+const ComboBoxTokenPicker = ({
```

Comment:
Was this file moved? What was the name? Diff isn't showing.

Reply:
Lol seems to have been a copy-paste error from before my time. I'll fix it though